### DISCUSSION TITLE
비밀 댓글 관련 이슈를 해결해 봅시다.

### DISCUSSION THREAD

**Discussion by pjy1368** [darass-team/2021-darass#780] [Ideas]
## [현재 댓글 UI]
![image](https://user-images.githubusercontent.com/56083021/138815436-acadec1d-ea8b-45ec-ab89-2b207a311b85.png)

## [문제점]
1. 비밀 댓글 작성자의 닉네임만 봐서는 비로그인 유저인지, 로그인 유저인지 알 수가 없다. 혹은 구분이 잘 가지 않는다.
2. 댓글 우측 상단 . . . 버튼을 누르면 댓글 열람 버튼이 뜨는데, 어떤 댓글에는 이 기능이 있고 어떤 댓글에는 이 기능이 없어서 헷갈린다. (이 부분은 1번이 확실하게 해결되면 괜찮아 질듯??)
3. 비로그인 유저 비밀 댓글이 열람되면, 비밀 대댓글까지 전부 공개된다는 사실을 유저가 알기 어렵다.

많은 의견 댓글 바랍니다.

**Comment by pjy1368** [comment]
[1번 UI 문제가 해결된다는 가정 하에]
3번은 비로그인 유저가 작성한 루트 비밀댓글에 대해서 툴팁을 적어두는 건 어떨까요? 예를 들어, "해당 비밀 댓글 열람에 성공하면, 모든 비밀 대댓글이 공개됩니다."

**Comment by jaeseongDev** [comment]
1번 - 블로그의 댓글 특성상 익명 유저와 비로그인 유저를 따로 구분할 필요는 크게 없어도 문제되지 않지 않을까라는 생각이 들어요! 그래서 현 체제도 충분히 괜찮지 않나라는 생각이 들었습니다!! 
혹시 구분을 해야하는 필요성이 든 이유가 있을까요 ? 

3번 - 툴팁을 활용해서 안내를 해도 괜찮을 것 같다는 생각이 드네요. 그리고 또 다른 생각으로써 지금 현재 상태로 두고, 비밀 댓글을 보는 불편함을 사용자로부터 살짝 느끼게 해서 로그인을 유도하는 것도 괜찮다는 생각이 들어요!

**Comment by jujubebat** [comment]
1번 : 비밀 댓글 작성자가 비로그인 유저인지 로그인 유저인지 알아야할 필요가 있을까요??

**Comment by zereight** [comment]
1. 구별하고 싶다면 
  - 카카오 소셜 로그인 유저는 프로필에 카카오, 네이버 소셜 로그인 유저는 네이버 프로필을 박거나
  - 따로 댓글 영역 내에 표식을 새기거나 (색상이든 뭐든)
  해야할것 같구요!
2. 저는 지금도 좋지만, 헷갈린다면 모든 댓글에 "..." UI 노출시킨다음 버튼을 눌렀을때 "권한이 없습니다." 얼럿 띄우는 방법이 있겠습니당
3. 이거는 툴팁위치를 어케하지

**Comment by pjy1368** [comment]
@jaeseongDev @jujubebat 
피드백해 준 사람이 아무래도 어떤 댓글에는 '...' 표시가 있고 어떤 것은 없다 보니, 거기에 궁금증이 있었다고 합니다. 그래서 특정 비밀 댓글이 비로그인인지 로그인인지 볼 수 있으면 좋겠다고 했습니다.
근데 이야기를 들어보니까, 그 두 개를 억지로 구분하기보다는 차라리 '...' 표시를 넣어주되 "권한이 없습니다."를 띄워주는 편이 좋을 듯합니다.

**Reply by jujubebat** [reply]
동의 합니다.
그냥 모든 비밀 댓글에 '...' 표시 넣어주고, 비밀번호 입력 가능하게 해도 괜찮을 것 같아요.
